madmom/ml/nn/layers.py

```diff
@@ -413,7 +572,7 @@ def activate(self, data, reset_gate, prev):
         return self.activation_fn(out)
```

Comment:
This functionality looks very similar to what  the `Gate` class already provides. Do we really need a separate class for the GRU cell?